controller/src/main/java/io/pravega/controller/store/InMemoryScope.java

```diff
@@ -175,4 +165,7 @@ public void refresh() {
         return CompletableFuture.completedFuture(null);
     }
 
+    public KeyValueTable getKeyValueTable(String name) {
```

Comment:
kvTablesMap is guarded by $lock so add synchronized annotation for this method

Reply:
done